### ISSUE TITLE
blank screen on the iphone when using angular-payments

### ISSUE THREAD

**Issue by noeltimothy** [laurihy/angular-payments#115]
I've been successfully using the stripe via an ionic popup form. However, once I moved to the live Stripe setting instead of test, my app just appears blank on ios.

Should I load my form from a https site for the live stuff?

**Comment by tucoinfo** [comment]
Yes I believe this is now mandatory. Indeed, from the current Stripe Security Docs: 

> TLS is only required for live transactions, and you can test your integration without using TLS. Once you’re ready to accept live charges, you will then need to set up TLS.

Personally I use Letsencrypt as I don't need extended validation.

**Comment by noeltimothy** [comment]
Thanks for the reply. I got the form working by loading it off https. However, now I have the following issue.

My ionic app has the following:
angular-payments
stripeCallback

My https form on my remote server has:
stripe.js
stripe.setpublishablekey(xx)
stripe-form

The problem is the stripeCallback on my app doesn't have a reference to the stripe object with the published key. If I do the setpublishablekey call inside my ionic app, I get the "stripe live error saying it's insecure". 

I can't implement the callback on the https server since the server doesn't have the entire angular framework and scopes.

**Comment by noeltimothy** [comment]
An update on this issue. 
Stripe requires stripe.js to be served from a https server. 
Ionic Popup and ionic modal failed to load external javascript. 

Looks like there needs to be a widget that loads external javascript and allows external javascript variables to be accessible from an angular controller.
